docs/acceptance.md

```diff
@@ -1,59 +1,99 @@
+  - [ ] It should deemphasize search UI
+  - [ ] It should link to a feedback form/survey 
+    - [ ] *TBD: CAN WE STICK WITH SURVEY GIZMO* confirm with *ellee mirwin*
+  - [ ] *STRETCH* It should include pagination *NEW FEATURE*
+  - [ ] *STRETCH* It should be fully responsive down to 320px
+
+## Shot Owner Detail Page UI (SERVER)
+- [ ] It should match the final spec provided by the UX team
+  - [ ] It should be fully responsive
+  - [ ] It should include a loading animation until images trigger onload callbacks *NEW FEATURE*
+  - [ ] it should include all current controls with the share panel minimized by default
+  
+## Non-Owner Shot Detail Page UI (SERVER)
+  - [ ] It should have three layout options that promote Page Shot for non-owner views *NEW FEATURE*
+    - [ ] Top banner
+    - [ ] Side banner
+    - [ ] Bottom banner
+    - [ ] It should provide context specific messaging for desktop browsers
+    - [ ] It should provide context specific messaging for non-ff mobile browsers
+    - [ ] It should provide context specific messaging for mobile-ff browsers
+  - [ ] It should include Firefox + Page Shot branding
+  - [ ] It should not have flagging UI in the nav bar
+  - [ ] It should have flagging UI in the footer
+
+## Owner Expired Shot page UI
+- [ ] It should match the final spec provided by the UX team
+
+## Non-Owner Specific Shot Detail Page UI
+- [ ] It should include promotional materials describing the utility of Page Shot and link to the landing page
+
+## Landing Page UI
+- [ ] It should match the final spec provided by the UX team
+- [ ] It should describe the value proposition of Page Shot
+- [ ] It should specify that Page Shot is currently only available for Firefox on Desktop for non-FF users
+- [ ] It should specify that Page Shot is ‘Coming Soon’ for Chrome users
+- [ ] It should specify for users of older versions of Firefox that they simply need to update Firefox to use it
+- [ ] *STRETCH:* it should let users of Firefox 54 + trigger the add-on onboarding flow from the site
+
+## Onboarding UI
+- [ ] It should match the final spec provided by the UX team
+- [ ] It should proceed stepwise to explain the basics of the page shot feature
+- [ ] It should initially suggest itself by adding a page shot icon to the tool bar and badging it
+- [ ] *STRETCH* If a user has not engaged with the badged toolbar icon in three weeks, we should pop a tab asking the user if they would like to learn about the new feature
+
+## New Measurements & Dashboards
+
+## Measures
+- [ ] Reporting of all measures should be derived from the state of the Telemetry flag in the browser.
+- [ ] All measures currently in the product should be audited to ensure they’re working as intended.
+- [ ] Onboarding UI should be instrumented to measure interaction rate w/buttons/controls etc.
+- [ ] Toolbar icon should record a special event if clicked while badged & clicked to onboard
+  - [ ] proposal: `addon/start-shot/onboarding-badge`
```

Comment:
We could also use a Custom Dimension.  Or we could do two events, one generic, and another specific.